docs/source/telegram.animation.rst

```diff
@@ -1,4 +1,4 @@
-:github_url: https://github.com/python-telegram-bot/python-telegram-bot/blob/master/telegram/files/animation.py
+:github_url: https://github.com/python-telegram-bot/python-telegram-bot/blob/v13.x/telegram/files/animation.py
```

Comment:
same. if those changes are not individual commits, e.g. hard to revert & cherry pick, I'm also okay with keeping as is.